### PR TITLE
add redirect for bgilstrap.

### PR DESCRIPTION
**Pull request by icebrg**

### CODE REVIEW

File: people/.htaccess [hunk 1]
```diff
@@ -4,6 +4,7 @@ RewriteRule ^agbeltran$ http://newt.oerc.ox.ac.uk/people/agb.rdf [R=303,L]
 RewriteRule ^aidan$ http://aidan.droppages.com/ [R=302,L]
 RewriteRule ^amy$ http://bosatsu.net/foaf/uberconf/amy.rdf [R=303,L]
 RewriteRule ^asinesio$ http://asinesio.github.io/foaf.rdf [R=303,L]
+RewriteRule ^bgilstrap$ http://bosatsu.net/foaf/brian.rdf://dl.dropboxusercontent.com/u/3128671/brian.html [R=303,L]
```
Comment (davidlehn) [inline people/.htaccess]: Did you mean this to be https://bosatsu.net/foaf/brian.rdf://dl.dropboxusercontent.com/u/3128671/brian.html?

Comment (davidlehn) [inline people/.htaccess]: Sorry, copy & paste error, I meant:
https://dl.dropboxusercontent.com/u/3128671/brian.html

Comment (icebrg) [inline people/.htaccess]: For the moment, I was just trying to get something super simple working to prove I could make it work. So my goal was to link to a simple html file as the starting point for my identity until I create something more useful.

Should I approach this differently?

Comment (davidlehn) [inline people/.htaccess]: That approach is fine, but you need to fix the URL to not have bsletten's URL as a prefix.

Comment (icebrg) [inline people/.htaccess]: Doh! Sorry. Fixing...

File: people/.htaccess [hunk 2]
```diff
@@ -4,6 +4,7 @@ RewriteRule ^agbeltran$ http://newt.oerc.ox.ac.uk/people/agb.rdf [R=303,L]
 RewriteRule ^aidan$ http://aidan.droppages.com/ [R=302,L]
 RewriteRule ^amy$ http://bosatsu.net/foaf/uberconf/amy.rdf [R=303,L]
 RewriteRule ^asinesio$ http://asinesio.github.io/foaf.rdf [R=303,L]
+RewriteRule ^bgilstrap$ http://dl.dropboxusercontent.com/u/3128671/brian.html [R=303,L]
```
Comment (davidlehn) [inline people/.htaccess]: You should probably just use https since dropbox will redirect anyway.

**Comment by icebrg** [comment]
Is the failed check something I introduced? Doesn't seem to be but not sure if I'm reading things correctly.

**Comment by davidlehn** [comment]
no, testing system is messed up with https links for some reason.

**Pull request merged**